Markdown: add "open in new window" icon and alt (#3817)
microsoft/BotFramework-WebChat: CHANGELOG.md

## [Unreleased]

### Added

> -  Resolves [#3087](https://github.com/microsoft/BotFramework-WebChat/issues/3087). External links in Markdown will now appended with an "open in new window" icon and text for accessibility, by [@compulim](https://github.com/compulim) in PR [#3817](https://github.com/microsoft/BotFramework-WebChat/pull/3817)

Reviewer: ```suggestion
-  Resolves [#3087](https://github.com/microsoft/BotFramework-WebChat/issues/3087). External links in Markdown will now be appended with an "open in new window" icon and accessibility label, by [@compulim](https://github.com/compulim) in PR [#3817](https://github.com/microsoft/BotFramework-WebChat/pull/3817)
```
